susemanager/src/mgr_bootstrap_data.py

```diff
@@ -353,6 +353,34 @@
     "libunwind",
 ]
 
+PKGLISTUBUNTU = [
+    "python3-tornado",
```

Comment:
Python 3? I thought salt is compiled for python 2 in Ubuntu. Please check also if the list is the same for 16.04 and 18.04

Reply:
Package lists are now separate for 16.04 and 18.04.